### ISSUE TITLE
Project gone stale?

### ISSUE THREAD

**Issue by MGlolenstine** [thlorenz/rid#63]
What's the state of this project?
It seems to have gone stale.

I can help out with some roadblocks, if that's what's holding the project back.

**Comment by thlorenz** [comment]
Thanks for the clear words @enzotar. I've pushed this off for too long it seems.
However I listened to what you said and for now just open source the repo as that was most pressing and also quickest to accomplish.

I'll clean up the text regarding the sponsors and so at a later time.

I will keep write access for anyone who contributed code in the past in the hopes that the project moves faster that way.

If any one else wants to keep write access please LMK until next week @contributors.

**Comment by p4-k4** [comment]
> Thanks for the clear words @enzotar. I've pushed this off for too long it seems. However I listened to what you said and for now just open source the repo as that was most pressing and also quickest to accomplish.
> 
> I'll clean up the text regarding the sponsors and so at a later time.
> 
> I will keep write access for anyone who contributed code in the past in the hopes that the project moves faster that way.
> 
> If any one else wants to keep write access please LMK.

Bravo, I truly hope this project can get the lift it deserves. It would be a shame to let your work go to waste by further restricting it, until of course you have some time. Would still recommend highlighting on the website/GH/YouTube the need for extra contributions asap. Cheers.

**Comment by John0x** [comment]
You should probably post this on reddit etc.
Just noticed this being open now on accident :'D

**Comment by thlorenz** [comment]
Yeah good point @John0x. I just wanna take care of all the other tasks I mentioned above first.
